ffn/core.py

```diff
@@ -1509,7 +1516,7 @@ def _erc_weights_ccd(x0,
         Griveau-Billion, Theophile and Richard, Jean-Charles and Roncalli,
         Thierry, A Fast Algorithm for Computing High-Dimensional Risk Parity
         Portfolios (2013).
-        Available at SSRN: https://ssrn.com/abstract=2325255
+        Available at SSRN: https://ssrn.com/abstract=23trading_days_per_year55
```

Comment:
Note, I assume you did a find and replace on 252 however, this one should remain. @oldrichsmejkal